lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2089,6 +2097,55 @@ def _enable_ssh_on_windows(self, node: Node) -> None:
         log.debug("SSH script result:")
         log.dump_json(logging.DEBUG, result)
 
+    def _get_blob_size(self, blob_url: str) -> int:
```

Comment:
Make the name more accurate, like `_get_vhd_os_disk_size`

Reply:
updated